.evergreen/.evg.yml

```diff
@@ -561,130 +576,157 @@ axes:
         variables:
            JDK: "jdk6"
 
+  # Choice of MongoDB storage engine
+  - id: storage-engine
+    display_name: Storage
+    values:
+      - id: mmapv1
+        display_name: MMAPv1
+        variables:
+           STORAGE_ENGINE: "mmapv1"
+      - id: wiredtiger
+        display_name: WiredTiger
+        variables:
+           STORAGE_ENGINE: "wiredtiger"
+      - id: inmemory
+        display_name: InMemory
+        variables:
+           STORAGE_ENGINE: "inmemory"
+           
 buildvariants:
 
 # Test packaging and other release related routines
 - name: static-checks
+  matrix_spec: { storage-engine: "wiredtiger" }
```

Comment:
I don't think this is needed but if it is can you explain why?

My hypotheses is it's only needed for the variants that actually require STORAGE_ENGINE to be defined (i.e. any that run tests that require mongo-orchestration

Reply:
I did not know that. I have removed it, and I have removed the `storage-engine` spec from all other tests that do not require it.